The sample code for the EventBridgeHandler implementation in the Scheduling Functions document is incorrect, causing TS2314

**Describe the content issue**:
The sample code for the EventBridgeHandler implementation in the Scheduling Functions document is incorrect, causing TS2314

```
amplify/hogejob/hogeschedule/handler.ts(3,23): error TS2314: Generic type 'EventBridgeHandler' requires 3 type argument(s).
amplify/hogejob/hogeschedule/handler.ts(3,51): error TS7006: Parameter 'event' implicitly has an 'any' type.
```

**URL page where content issue is**:
https://docs.amplify.aws/react/build-a-backend/functions/scheduling-functions/

--- a/src/pages/[platform]/build-a-backend/functions/scheduling-functions/index.mdx
+++ b/src/pages/[platform]/build-a-backend/functions/scheduling-functions/index.mdx
@@ -51,7 +51,7 @@ Their handlers can be typed using the `EventBridgeHandler` type:
 ```ts title="amplify/jobs/weekly-digest/handler.ts"
 import type { EventBridgeHandler } from "aws-lambda";
 
-export const handler: EventBridgeHandler = async (event) => {
+export const handler: EventBridgeHandler<"Scheduled Event", null, void> = async (event) => {
   console.log("event", JSON.stringify(event, null, 2))
 }
 ```

